cycode/cli/code_scanner.py

```diff
@@ -1099,6 +1112,21 @@ def _is_file_extension_supported(scan_type: str, filename: str) -> bool:
     return not filename.endswith(consts.SECRET_SCAN_FILE_EXTENSIONS_TO_IGNORE)
 
 
+def _is_iac(scan_type: str) -> bool:
+    return scan_type == consts.INFRA_CONFIGURATION_SCAN_TYPE
+
+
+def _is_tfplan_json_file(file: str, content: str) -> bool:
+    if not file.endswith('.json'):
+        return False
+    try:
+        tf_plan = json.loads(content)
```

Comment:
let's create a function that will load json and handle all errors. if errors have occurred the None will be returned. for example check "get_file_content" function